gradle-plugin-release-plugin/plugin/src/main/groovy/com/novoda/releaseplugin/GradleReleasePlugin.groovy

```diff
@@ -0,0 +1,33 @@
+package com.novoda.releaseplugin
+
+import com.novoda.releaseplugin.internal.ParseChangeLogTask
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class GradleReleasePlugin implements Plugin<Project> {
+
+    @Override
+    void apply(Project project) {
+        def pluginExtension = project.extensions.create('releasePlugin', GradleReleaseExtension, project)
+        configureChangelog(project, pluginExtension)
+    }
+
+    private static void configureChangelog(Project project, GradleReleaseExtension pluginExtension) {
+        def parseChangelog = project.tasks.create('parseReleaseChangelog', ParseChangeLogTask) { task ->
+            task.changelog.set(pluginExtension.changelog)
+            task.version.set(pluginExtension.version)
+            task.releaseChangelog = project.file("releaseChangelog.md")
```

Comment:
@mr-archano 
This is actually an issue. I'd like to create that file in the `build` directory, but when I try to do that `File.createNewFile()` throws a `FileNotFoundException`. The file gets create when the task is executed, could it be that the `build` directory doesn't exist at that time?
Any suggestion?